src/electos/ballotmaker/validate_edf.py

```diff
@@ -1,18 +1,31 @@
+import json
 import logging
+from dataclasses import asdict
 from pathlib import Path
 
-from electos.ballotmaker.election_data import ElectionData
+from electos.ballotmaker.constants import NO_ERRORS
+from electos.ballotmaker.data.extractor import BallotDataExtractor
 
 log = logging.getLogger(__name__)
 
 
+def report(data, **opts):
+    """Generate data needed by BallotLab."""
+    extractor = BallotDataExtractor()
+    ballot_data = extractor.extract(data)
+    ballot_data = [asdict(_) for _ in ballot_data]
+    print(json.dumps(ballot_data, indent=4))
+
+
 def validate_edf(
```

Comment:
At this point in it's evolution I think this function has changed enough that the name "validate_edf" is misleading. I would expect the command to tell me if the EDF itself is valid or not. But the output is extracted election data for ballots, with validation just a side effect.

No need to change it for milestone 2, just noting for later.

Reply:
What I'd like the `validate` command to do is just to run through the process of reading the EDF, then using the existing logging statements, just report what it finds in the EDF (contest count, etc.).

What do you think?

Reply:
That output sounds useful but it isn't validation. How about `summarize` or `summary`?

Reply:
That's a good idea -- but in fact the process, whatever it's called, will effectively validate the EDF. Since this is a new feature discussion, I created [a thread about the `validate` or `summarize` command](https://github.com/TrustTheVote-Project/BallotLab/discussions/23#discussioncomment-3691890) under [BallotMaker CLI commands · Discussion #23](https://github.com/TrustTheVote-Project/BallotLab/discussions/23)